docs/releases/changelog-dev.md

```diff
@@ -17,6 +17,10 @@
 
   [#747](https://github.com/qilimanjaro-tech/qililab/pull/747)
 
+- Added `threshold_rotations` argument to `compile()` method in `QProgram`. This argument allows to use rotation angles on measurement instructions if not specified. Currently used to use the angle rotations specified on the runcard (if any) so the user does not have to explicitly pass it as argument to the measure instruction.  Used for classification of results in Quantum Machines's modules.
```

Comment:
If this is a new addition to the runcard please show here an example on how to use it as a segment of the runcard (specifying its location)

Reply:
Done!